App.tsx

```diff
@@ -180,25 +194,25 @@ export const App: React.FC<AppProps> = ({ navigation }) => {
                 <InfoListItem
                     title={'South Hills Farm'}
                     subtitle={'Offline'}
-                    backgroundColor={theme.dark ? PXBColors.darkBlack[100] : 'white'}
-                    statusColor={PXBColors.orange[500]}
+                    backgroundColor={theme.colors.surface}
+                    statusColor={theme.dark ? PXBColors.red[200] : PXBColors.red[500]}
```

Comment:
For alarm status items, I would suggest using red[500], which is the standard status color, independent of the theme colors.

Reply:
Should the font color match the status color of red[500] for alert status items?

Reply:
Does this apply to alert icons in the ScoreCard as well?

Reply:
I think yes on the font, but you may want to check the accessibility. Generally, anywhere we are using the color to represent "Alarm" it should be 500. Things like errors and warnings can be 200, but "Alarms" has a very specific color definition from our status palette.